en/docs/develop-components/deploy-a-containerized-application.md

```diff
@@ -0,0 +1,119 @@
+# Deploy a Containerized Application 
+
+Using Choreo, you can easily deploy applications written in different language frameworks (such as Java, Go, NodeJs, Python, etc.) on shared or private data planes using containers.
+
+Choreo supports deploying containerized applications for the following component types:
+
+- Service
+- Scheduled Trigger
+- Manual Trigger
+- REST API
```

Comment:
Need to remove this since the REST API component type is no more.

Reply:
done